### PR TITLE
Add profile for gnome-ring

### PR DESCRIPTION
**Pull request by PharmaceuticalCobweb**
Added profile for gnome-ring in /etc. Pretty much the same as the Skype profile.

### CODE REVIEW

File: etc/gnome-ring.profile [hunk 1]
```diff
@@ -0,0 +1,32 @@
+# Firejail profile for gnome-ring
+# This file is overwritten after every install/update
+# Persistent local customizations
+include /etc/firejail/default.local
```
Comment (SpotComms) [inline etc/gnome-ring.profile]: default -> gnome-ring

File: etc/gnome-ring.profile [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+# Persistent global definitions
+include /etc/firejail/globals.local
+
+noblacklist ${HOME}/.local/share/gnome-ring
+
+include /etc/firejail/disable-common.inc
+include /etc/firejail/disable-devel.inc
+include /etc/firejail/disable-passwdmgr.inc
+include /etc/firejail/disable-programs.inc
+
```
Comment (SpotComms) [inline etc/gnome-ring.profile]: Does it work with `include /etc/firejail/whitelist-var-common.inc` ?

Comment (PharmaceuticalCobweb) [reply etc/gnome-ring.profile]: Had to add /etc/firejail/whitelist-var-common.inc to my /etc/ folder since I didn't have it. It does seem to work.

**Comment by SpotComms** [comment]
Merged, thanks!

**Pull request merged**